components/switch/switch-visibility.js

```diff
@@ -0,0 +1,22 @@
+import '../icons/icon.js';
+import { html, LitElement } from 'lit-element/lit-element.js';
+import { SwitchMixin } from './switch-mixin.js';
+
+class VisibilitySwitch extends SwitchMixin(LitElement) {
+
+	constructor() {
+		super();
+		this.label = 'Visibility';
```

Comment:
This needs to be languified - in some discussion with Design about the label.  Stay tuned.